src/index.html

```diff
@@ -2,15 +2,55 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     class="first_search-form"
+     method="get"
+     action="thing">
+
+      <div class="first">
+        <label for="big" data-qa="keypress">
+          <img
+           class="big_search"
+           src="images/Search.svg"
+           alt="Search icon">
```

Comment:
Please, fix this common mistake everywhere
```suggestion
          <img
             class="big_search"
             src="images/Search.svg"
             alt="Search icon"
           >
```